server/server.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express')
+const app = express()
+var mysql = require('mysql');
+const port = 3000
+const cors = require('cors')
+app.use(cors()); 
+
+//factor out this connection
+//import connection from db
+
+//start working on client side code
+//handle response to build graph
+//smallest success - react component fetches at URL returns data
+    //simple React component
+    //console.log(results) - of fetch
+    //build new Graph from data
+
+var db = mysql.createConnection({
+    host: "localhost",
+    user: "root",
+    password: "password",
+    database: "tradeDeskdb"
+});
+
+// app.get('/', (req, res) => res.send('Hello World!'))
```

Comment:
reminder to take out the stray code before making the PR

Reply:
Thank you! Will make changes.